dask_cuda/tests/test_initialize.py

```diff
@@ -214,3 +215,27 @@ def test_initialize_ucx_all(protocol):
     p.start()
     p.join()
     assert not p.exitcode
+
+
+def _test_dask_cuda_import(monkeypatch):
```

Comment:
@TomAugspurger @pentschev - I used a combination of your suggestions (thanks!)

Let me know what you think.

Reply:
I think this is good. The only behavior I'm a little worried about is what the behavior of a pickled `monkeypatch` is since we're sending it through the `mp.Process`. The [docs](https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch) suggest that you can use `pytest.MonkeyPatch.context()` if the fixture isn't available, so maybe I'd suggest that:

```
with pytest.MonkeyPatch.context() as monekypatch:
    ...
```

Only question is whether we'd want to test any functionality (like creating a LocalCUDACluster) beyond the imports.